pwndbg/commands/context.py

```diff
@@ -802,10 +801,15 @@ def context_backtrace(with_banner=True, target=sys.stdout, width=None):
     while True:
         prefix = bt_prefix if frame == this_frame else " " * len(bt_prefix)
         prefix = f" {c.prefix(prefix)}"
-        addrsz = c.address(pwndbg.ui.addrsz(frame.pc()))
-        symbol = c.symbol(pwndbg.gdblib.symbol.get(int(frame.pc())))
+        frame_pc = frame.pc()
+        frame_pc_on_stack = pwndbg.gdblib.stack.find_addr_on_stack(frame_pc)
+        padded_text = "0x" + hex(frame_pc_on_stack)[2:].zfill(pwndbg.gdblib.arch.ptrsize * 2)
```

Comment:
`hex(...)` returns a `0x...` string, so we don't need to `[2:]` it and prepend a `0x` string

Also nitpick, but we fetch and compute `pwndbg.gdblib.arch.ptrsize * 2` in each loop iteration :<

Reply:
>>> hex(1234).zfill(8)
'0000x4d2'

i was trying to avoid this - perhaps theres a way to zfill an already 0x prefixed string?

@disconnect3d

Reply:
will hoist the computation out of the loop

Reply:
I'm not sure about this zfill. Like, do we really want to display those zeroes? :P They are redundant